components/core/src/clp/ffi/Value.hpp

```diff
@@ -150,12 +150,10 @@ class Value {
     template <FundamentalPrimitiveValueType T>
     explicit Value(T value) : m_value{value} {}
 
-    // Disable copy constructor and assignment operator
-    Value(Value const&) = delete;
-    auto operator=(Value const&) -> Value& = delete;
-
-    // Default move constructor and assignment operator
+    // Default copy/move constructor and assignment operator
```

Comment:
Since everything except the constructor is default, we don't need to say they're default explicitly, right?